### PR TITLE
chore(codecov): Adjust Codecov parameters

### PR DESCRIPTION
**Pull request by armenzg**
Changes:
* Use new diff focused comments [1]
* If a comment has been deleted, do not create a new comment
* Bundle reporter to only comment when the bundle increases at least 10kbs

[1]
image

### CODE REVIEW

File: codecov.yml [hunk 1]
```diff
@@ -78,16 +78,17 @@ comment:
...
-  layout: 'diff, files'
-  # Update, if comment exists. Otherwise post new.
-  behavior: default
-  # Comments will only post when coverage changes. Furthermore, if a comment
-  # already exists, and a newer commit results in no coverage change for the
-  # entire pull, the comment will be deleted.
-  require_changes: true
+  # Show only the git diff / patch coverage, and exclude any comparison between base and head:
+  layout: 'condensed_header, condensed_files, condensed_footer'
+  hide_project_coverage: true
```
Comment (armenzg) [inline codecov.yml:83]: These two lines will enable the following:
image

File: codecov.yml [hunk 2]
```diff
@@ -78,16 +78,17 @@ comment:
...
+  # Update, if exists. Otherwise post new. Skip if deleted.
+  behavior: once
```
Comment (armenzg) [inline codecov.yml:85]: Explained [here](https://docs.codecov.com/docs/pull-request-comments#behavior).

image

File: codecov.yml [hunk 3]
```diff
@@ -78,16 +78,17 @@ comment:
...
+  # Comments will only post when coverage drops OR there are uncovered lines of code
+  require_changes: 'coverage_drop OR uncovered_patch'
   require_base: true # must have a base report to post
   require_head: true # must have a head report to post
   require_bundle_changes: True # only post a comment when there are bundle size changes
+  bundle_change_threshold: '10Kb'
```
Comment (armenzg) [inline codecov.yml:93]: I made a very small PR and it increased 7.19kb. This should be a good starting point to only report when unwanted increases happen.
image

**Review by dashed — APPROVED** [review]

**Review by priscilawebdev — APPROVED** [review]
thanks for doing this 🥳

**Pull request merged**